pkg/utils/utils.go

```diff
@@ -291,12 +291,16 @@ func configSriovDevice(iface *sriovnetworkv1.Interface, ifaceStatus *sriovnetwor
 				if err = setVfGuid(addr, pfLink); err != nil {
 					return err
 				}
+			} else if driver != "" {
```

Comment:
why is this needed ? to avoid triggering the WA  ?

Reply:
I will use this instead.

https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/245

This is needed if there was a problem with one of the vfs and some of them are in vfio we will try to see check if VfIsReady and that will always fail because the VF is already attached to a user-space driver